qt/CMakeLists.txt

```diff
@@ -236,14 +236,29 @@ set_target_properties(
 
 tr_win32_app_info(${TR_NAME}-qt
     "Transmission Qt Client"
-    "transmission-qt"
-    "transmission-qt.exe"
-    "qtr.ico")
+    "${TR_NAME}-qt"
+    "${TR_NAME}-qt.exe"
+    "${TR_NAME}.ico")
```

Comment:
This is the change I'm the most concerned about. I'm not really sure how this works. I tried to compare with the gtk file, but they both seem to do something different, and unless I'm missing something, one of them is broken currently (likely the qt one).

The way I understand it currently is what this name should be is the same file referenced in the WiX installer, which I have updated to point to the same icon in the new shared location. If I'm wrong on this, please let me know how this should work and I can patch it!

Thanks!

Reply:
I'm assuming this is okay now and resolving this thread. Feel free to comment and re-open if there are any concerns.